scripts/install_python.sh

```diff
@@ -22,13 +22,25 @@ apt_install \
     python3-venv \
     swig
 
+# Setup a virtualenv to install things into
+
+# Put things under /opt/venv, if nothing else is specified
+export VIRTUAL_ENV="${VIRTUAL_ENV:=/opt/venv}"
+export PATH="${VIRTUAL_ENV}/bin:${PATH}"
+
+# Make sure that Rstudio sees these env vars too
+echo "PATH=${PATH}" >>"${R_HOME}/etc/Renviron.site"
+echo "VIRTUAL_ENV=${VIRTUAL_ENV}" >>"${R_HOME}/etc/Renviron.site"
+
+python3 -m venv "${VIRTUAL_ENV}"
+
+# Upgrade version of pip inside the virtualenv
 python3 -m pip --no-cache-dir install --upgrade \
     pip
 
-# Some TF tools expect a "python" binary
-if [ ! -e /usr/local/bin/python ]; then
-    ln -s "$(which python3)" /usr/local/bin/python
-fi
+# Make the venv owned by the staff group, so users can install packages
+# without having to be root
+chown -R rstudio:staff /opt/venv
```

Comment:
Don't hard-code the username. There is no guarantee that the user exists.

Reply:
@eitsupi in `install_texlive.sh` I see:

```
NON_ROOT_USER=$(getent passwd "1000" | cut -d: -f1)
```

and this is used as username. Should I do that?

Reply:
Is there a bad reason not to be the root user?
If so, please do so.

However, in that case, I don't think it will work if the uid is anything other than 1000

Reply:
Ok, given I was trying to replicate what happens in R, I've set this to match what is done to `$R_HOME/site-library`